src/offcanvas/offcanvas-stack.ts

```diff
@@ -0,0 +1,201 @@
+import {DOCUMENT} from '@angular/common';
+import {
+  ApplicationRef,
+  ComponentFactoryResolver,
+  ComponentRef,
+  EventEmitter,
+  Inject,
+  Injectable,
+  Injector,
+  NgZone,
+  RendererFactory2,
+  TemplateRef
+} from '@angular/core';
+import {finalize, Subject} from 'rxjs';
+
+import {ngbFocusTrap} from '../util/focus-trap';
+import {ContentRef} from '../util/popup';
+import {ScrollBar} from '../util/scrollbar';
+import {isDefined, isString} from '../util/util';
```

Comment:
```suggestion
import {isString} from '../util/util';
```
if `??=` change assignments are accepted, this has to be removed to prevent linting failure

Reply:
disregards, misread intent in other place